### PR TITLE
Unify cases nav links - Fix #3211

### LINKED ISSUE #3211: Navigation bar institutes level slight inconsistency
The most common way to represent the institutes level is taken from the Cases page view:
Screenshot 2022-03-04 at 09 49 45
But for e.g. the variantS views we have:
Screenshot 2022-03-04 at 09 49 49

It's not necessarily bad to shrink the nav bar size once travelling out the tree, but there is also no real logic to which pages have which version.

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Remove the extra subscript "cases" from the institute display name on the navbar from some of the pages to unify.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/institutes/templates/overview/cases.html [hunk 1]
```diff
@@ -11,8 +11,11 @@
   <li class="nav-item">
     <a class="nav-link" href="{{ url_for('cases.index') }}">Institutes</a>
   </li>
+  <li class="nav-item">
+    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">{{ institute.display_name }}</a>
+  </li>
   <li class="nav-item active">
-    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">{{ institute.display_name }} cases </a>
+    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">Cases</a>
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/cases.html:18]: This is the only place it makes perfect sense to say "cases" explicitly.

File: scout/server/blueprints/institutes/templates/overview/causatives.html [hunk 2]
```diff
@@ -18,13 +18,48 @@
...
   <li class="nav-item">
-    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">{{ institute.display_name }} cases </a>
+    <a class="nav-link" href="{{ url_for('overview.cases', institute_id=institute._id) }}">{{ institute.display_name }}</a>
   </li>
   <li class="nav-item active">
     <span class="navbar-text">Causative variants</span>
   </li>
 {% endblock %}
 
+{% block content_main %}
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/causatives.html:28]: Sorry for the edits! I moved the macro to the end to make the block structure more evident, without changing the content.

File: scout/server/blueprints/institutes/templates/overview/causatives.html [hunk 3]
```diff
@@ -18,13 +18,48 @@
...
+<div class="container-float">
+  <div class="row" id="body-row"> <!--sidebar and main container are on the same row-->
+    {{ institute_actionbar(institute) }} <!-- This is the sidebar -->
+    <div class="col">
```
Comment (dnil) [inline scout/server/blueprints/institutes/templates/overview/causatives.html:32]: Well, almost. That container float made no sense to me. 😊

File: scout/server/blueprints/institutes/templates/overview/causatives.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/institutes/templates/overview/causatives.html:28]: It's fine, just let us know when you are done if you need a review!

Comment (dnil) [reply scout/server/blueprints/institutes/templates/overview/causatives.html:28]: Anytime!

**Review by northwestwitch — APPROVED** [review]
Tested locally and looks good! 💯

**Pull request merged**